lib/note-list/index.jsx

```diff
@@ -515,7 +485,8 @@ const mapStateToProps = ({ appState: state, settings: { noteDisplay } }) => {
    *
    * @type {String} preview excerpt for the current note
    */
-  const selectedNoteTitle = selectedNote && getNoteTitle(selectedNote);
+  const selectedNoteTitle =
+    selectedNote && getNoteTitleAndPreview(selectedNote).preview;
```

Comment:
Should this const be called `selectedNoteTitleAndPreview`? It's a little confusing that it is getting the `.preview` value but is called `selectedNoteTitle`.

Reply:
Good catch, thanks!